src/backend/backend.nim

```diff
@@ -29,6 +29,9 @@ type
     name* {.serializedFieldName("name").}: string
     address* {.serializedFieldName("address").}: string
     favourite* {.serializedFieldName("favourite").}: bool
+    chainShortNames* {.serializedFieldName("chainShortNames").}: string
```

Comment:
@IvanBelyakoff in general for all properties here we don't need `serializedFieldName` pragma.